datalad_registry_client/tests/test_submit.py

```diff
@@ -93,7 +120,7 @@ def test_submit_explicit_endpoint(tmp_path):
     assert_in_results(
         ds.registry_submit(url=ds.path, endpoint="abc", on_failure="ignore"),
         action="registry-submit", type="dataset",
-        path=ds.path, status="error")
+        path=ds.path, url=ds.path, status="error")
```

Comment:
note to myself: ATM allows for URL to just be a local path.  Could be a "feature" I guess for some local deployments to keep a "catalog" of available (locally) datasets.